binrz/man/rz-find.1

```diff
@@ -1,66 +1,70 @@
-.Dd Jun 24, 2020
+.Dd Jan 24, 2024
 .Dt RZ_FIND 1
 .Sh NAME
 .Nm rz-find
 .Nd Advanced commandline hexadecimal editor
```

Comment:
I think the description is wrong, it's not an editor

Reply:
"Find byte patterns in files". how is this?

Reply:
![image](https://github.com/rizinorg/rizin/assets/62786136/77a27fcc-34b4-4b86-977d-0307638db0ac)

Reply:
![image](https://github.com/rizinorg/rizin/assets/62786136/16909f68-94b4-44de-b031-6a7436e8cb86)

Reply:
Yes, that would work